### PR TITLE
Indicator for QR code scan, Larger camera height if supported

### PR DESCRIPTION
**Pull request by gabriel**
If QR code was scanned puts an activity indicator in its place.

Using larger camera height for taller phones. I was reluctant to choose a height based on dimension instead of flex: 1, but we need to specify a fixed height for the camera component.

2017-05-16 at 4 00 pm

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/react-hackers

**Comment by keybase-ci-visdiff** [comment]
The commits f9b194de6a6d...c6f2c0ba8ebf introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Code_Page-Scanned_Code_Mobile**  
   Code_Page-Scanned_Code_Mobile rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 1d50a4e5a077...5f40d2617818 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Code_Page-Scanned_Code_Mobile**  
   Code_Page-Scanned_Code_Mobile rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 1d50a4e5a077...38ff2ac1d635 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Code_Page-Scanned_Code_Mobile**  
   Code_Page-Scanned_Code_Mobile rendered
</summary>

File: shared/login/register/code-page/index.render.native.js [hunk 1]
```diff
@@ -29,6 +29,13 @@ class CodePageRender extends Component<void, Props, void> {
   }
 
   renderScanCode () {
+    if (this.props.qrCodeScanned) {
+      return (
+        <Box style={{...stylesQRScan, flex: 1, flexDirection: 'row', alignItems: 'center', justifyContent: 'center'}}>
+          <NativeActivityIndicator size='large' />
```
Comment (chrisnojima) [inline shared/login/register/code-page/index.render.native.js]: we need something more descriptive than this. This makes it seem like its waiting for something like processing etc, its not clear you actually need to do something on a different computer

**Comment by keybase-ci-visdiff** [comment]
The commits 8f0c4dbac823...a696770fc025 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Code_Page-Scanned_Code_Mobile**  
   Code_Page-Scanned_Code_Mobile rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 8f0c4dbac823...4f1f0fc270c9 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Code_Page-Scanned_Code_Mobile**  
   Code_Page-Scanned_Code_Mobile rendered
</summary>

**Comment by gabriel** [comment]
If my device is existing iPhone will show message:
"You should follow the instructions on the other device to continue."

**Comment by keybase-ci-visdiff** [comment]
The commits 8f0c4dbac823...2f0beb3b7605 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Code_Page-Scanned_Code_Mobile**  
   Code_Page-Scanned_Code_Mobile rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 404ff4ec4b57...eb67b38f45c6 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Code_Page-Scanned_Code_Mobile**  
   Code_Page-Scanned_Code_Mobile rendered
</summary>

**Pull request merged**